webapp/tests/elster_client/test_elster_client.py

```diff
@@ -703,7 +714,8 @@ def test_not_known_key_results_in_unchanged_value(self):
         with patch('app.elster_client.elster_client.current_user', MagicMock(is_active=True, is_authenticated=True)):
             result = _generate_est_request_data(unknown_keys)
 
-        self.assertEqual(unknown_keys, result['est_data'])
```

Comment:
Why does this not work anymore? (I can't mark the complete test, sorry)

Reply:
Because we do now set `person_a_has_merkzeichen_bl` and so on. Therefore the resulting data is not just the input data but the input data plus the merkzeichen keys.